pywinauto/hooks.py

```diff
@@ -262,11 +261,17 @@ def mouse_low_level_handler(code, event_code, kb_data_ptr):
 
     def unhook_mouse(self):
         """Unhook mouse events"""
+        if not self.mouse_is_hook:
```

Comment:
`if self.mouse_is_hook:` and then 2 lines is shorter. Empty `return` is not needed then.